2022/40xxx/CVE-2022-40626.json

```diff
@@ -3,16 +3,118 @@
     "data_format": "MITRE",
```

Comment:
'Zabbix' does not match '^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$'